concepts/delta_query_overview.md

```diff
@@ -35,36 +35,41 @@ If a client uses a query parameter, it must be specified in the initial request.
 
--   If a `$select` query parameter is used, the parameter indicates that the client prefers to only track changes on the properties or relationships specified in the `$select` statement. If a change occurs to a property that is not selected, the resource for which that property changed will not appear in the delta response after a subsequent request.
+-   If a `$select` query parameter is used, the parameter indicates that the client prefers to only track changes on the properties or relationships specified in the `$select` statement. If a change occurs to a property that is not selected, the resource for which that property changed does not appear in the delta response after a subsequent request.
 -   `$expand` is not supported.
 
+For users and groups beta (preview) APIs, scoping filters allow you to track changes to one or more specific users or groups by objectId. For example, the following request: https://graph.microsoft-ppe.com/beta/groups/delta/?$filter= id eq '477e9fc6-5de7-4406-bb2a-7e5c83c9ae5f' or id eq '004d6a07-fe70-4b92-add5-e6e37b8acd8e' returns changes for the groups matching the ids specified in the query filter. 
+
 ## Resource representation in the delta query response
 
 -   Newly created instances of a supported resource are represented in the delta query response using their standard representation.
 
 -   Updated instances are represented by their **id** with *at least* the properties that have been updated, but additional properties may be included.
 
--   Changes to relationships on users and groups are represented as annotations on the standard resource representation. These annotations use the format `propertyName@delta`, 
-and only appear when the client explicitly chooses to track changes to the relationship by using the `$select` parameter.
+-   Changes to relationships on users and groups are represented as annotations on the standard resource representation. These annotations use the format `propertyName@delta`. The annotations are included in the response of the initial delta query request. For tracked changes, the annotations only appear when the client explicitly chooses to track changes to the relationship by using the `$select` parameter.
+
+Removed instances are represented by their **id** and an `@removed` object. The `@removed` object may include additional information about why the instance was removed. For example,  "@removed": {"reason": “changed”}.
 
--   Removed instances are represented using only their **id** and an `@removed` node. The `@removed` node may include additional information about why the instance was removed.
+Possible @removed reasons can be *changed* or *deleted*.
+- *Changed* indicates the item was deleted and may be restored from [deletedItems](../api-reference/beta/resources/directory.md).
+- *Deleted* indicates the item is [permanently deleted](../api-reference/beta/api/directory_deleteditems_delete.md).
```

Comment:
I don't necessarily think what you have here needs to be changed, but its worth thinking about how developers will interpret this.